index.html

```diff
@@ -93,61 +93,70 @@ <h2 class="visually-hidden">Навигация по сайту</h2>
               </li>
             </ul>
           </div>
-          <section class="page-nav-container">
-            <h2 class="visually-hidden">Каталог товаров</h2>
-            <div class="nav-catalog">
-              <a class="nav-catalog-link" href="catalog.html">
-                каталог товаров
-              </a>
-              <button class="btn-catalog" type="button" tabindex="-1"></button>
-            </div>
-            <div class="catalog-group">
-              <h2 class="visually-hidden">Категории товаров</h2>
-              <div class="catalog-group-inner">
-                <ul class="other-list">
-                  <li class="other-item">
-                    <a class="other-link" href="#">Виртуальная реальность </a>
-                  </li>
-                  <li class="other-item">
-                    <a class="other-link" href="#">Моноподы для cелфи </a>
-                  </li>
-                  <li class="other-item">
-                    <a class="other-link" href="#">Экшн-камеры</a>
-                  </li>
-                </ul>
-                <ul class="clock-list">
-                  <li class="clock-item">
-                    <a class="clock-link" href="catalog.html"
-                      >Фитнес-браслеты</a
-                    >
-                  </li>
-                  <li class="clock-item">
-                    <a class="clock-link" href="catalog.html">Умные часы</a>
-                  </li>
-                </ul>
-                <ul class="quadcopters-list">
-                  <li class="quadcopters-item">
-                    <a class="quadcopters-link" href="catalog.html"
-                      >Фитнес-браслеты
-                    </a>
-                  </li>
-                </ul>
-              </div>
-            </div>
+          <div class="page-nav-container">
             <nav class="nav-page">
+              <div class="nav-catalog">
+                <a class="nav-page__link link-uppercase" href="catalog.html">
+                  каталог товаров
+                </a>
+                <button
+                  class="btn-catalog"
+                  type="button"
+                  tabindex="-1"
+                ></button>
+              </div>
+              <div class="catalog-group">
+                <h2 class="visually-hidden">Категории товаров</h2>
+                <div class="catalog-group-inner">
+                  <ul class="other-list">
+                    <li class="catalog-group-item">
+                      <a class="catalog-group-link" href="#"
+                        >Виртуальная реальность
+                      </a>
```

Comment:
```suggestion
                      <a class="catalog-group-link" href="#">
                         Виртуальная реальность
                      </a>
```
С редактором надо что-то сделать чтобы он эти скобки не переводил на другую строку